src/user/views/user_views.py

```diff
@@ -124,6 +124,29 @@ def referral_rsc(self, request):
 
         return Response({"amount": amount})
 
+    @action(
+        detail=True,
+        methods=[RequestMethods.POST],
+        permission_classes=[RequestorIsOwnUser],
+    )
+    def set_should_display_rsc_balance(self, request, pk=None):
+        try:
+            user = User.objects.get(id=request.user.id)
+            target_value = request.data.get("should_display_rsc_balance_home")
```

Comment:
Should we have some input check here? To ensure that it is a bool

Reply:
i think it's ok as is. If it's not a bool, it will throw an error because of the model field type.